src/Pods/Admin/Config/Pod.php

```diff
@@ -941,11 +943,12 @@ public function get_fields( \Pods\Whatsit\Pod $pod, array $tabs ) {
 					'boolean_yes_label' => '',
 				],
 				'rewrite_custom_slug'    => [
-					'label'      => __( 'Custom Rewrite Slug', 'pods' ),
-					'help'       => __( 'help', 'pods' ),
-					'type'       => 'text',
-					'default'    => '',
-					'depends-on' => [ 'rewrite' => true ],
+					'label'         => __( 'Custom Rewrite Slug', 'pods' ),
+					'help'          => __( 'help', 'pods' ),
+					'type'          => 'slug',
+					'slug_fallback' => '-',
+					'default'       => '',
+					'depends-on'    => [ 'rewrite' => true ],
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 5660c19fdac56c605f6549c4d8cd6da1</sub>